### PR TITLE
AR > Page > Redirection, template, title.

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://moderntribe.atlassian.net/browse/ETP-42
🎥 https://www.loom.com/share/6759f34b44b64829b1d064805311a97d

🗒 Notes

- The templates were fine, we need to add some compatibility fixes for supported themes. Like the one I added for twentytwenty.

### CODE REVIEW

File: src/Tribe/Attendee_Registration/Main.php [hunk 1]
```diff
@@ -42,16 +42,16 @@ public function get_slug() {
...
+				empty( $page )
+				&& has_shortcode( $page->post_content, 'tribe_attendee_registration' )
 			)
 		) {
-			$slug = Tribe__Settings_Manager::get_option( 'ticket-attendee-info-slug', $this->default_page_slug );
+			return $slug;
 		}
 
-		return $slug;
+		return Tribe__Settings_Manager::get_option( 'ticket-attendee-info-slug', $this->default_page_slug );
```
Comment (bordoni) [inline src/Tribe/Attendee_Registration/Main.php]: Can we use `tribe_get_option` here?

File: src/resources/postcss/tickets-registration-page.pcss [hunk 2]
```diff
@@ -379,6 +379,17 @@ body.page-tribe-attendee-registration {
...
 	}
+
+	/* -------------------------------------------------------------------------
+	 * Theme Overrides - Twenty twenty
+	 * ------------------------------------------------------------------------- */
+
+	&.tribe-theme-twentytwenty {
+
+		.entry-content .tribe-tickets__registration {
+			max-width: 960px !important;
```
Comment (bordoni) [inline src/resources/postcss/tickets-registration-page.pcss]: We really can't do it with a combo of ID + Classes + Elements? If not it's ok.

Comment (juanfra) [reply src/resources/postcss/tickets-registration-page.pcss]: No, because they rely on the `:not()` and it's taking over.

**Comment by bordoni** [comment]
There seems to be some fails here on the testing, lets make sure we fix it.

File: src/Tribe/Attendee_Registration/Template.php [hunk 3]
```diff
@@ -352,13 +352,18 @@ public function modify_archive_title( $title ) {
 	 * @return string
 	 */
 	public function get_page_title() {
+		$title = esc_html__( 'Attendee Registration', 'event-tickets' );
```
Comment (sc0ttkclark) [inline src/Tribe/Attendee_Registration/Template.php]:
```suggestion
		$title = __( 'Attendee Registration', 'event-tickets' );
```

File: src/Tribe/Attendee_Registration/Template.php [hunk 4]
```diff
@@ -352,13 +352,18 @@ public function modify_archive_title( $title ) {
...
+
 		/**
 		 * `tribe_tickets_attendee_registration_page_title`
 		 * Filters the attendee registration page title
 		 *
-		 * @param string the "Attendee Registration" title
+		 * @param string the "Attendee Registration" page title.
 		 */
-		return apply_filters( 'tribe_tickets_attendee_registration_page_title', esc_html__( 'Attendee Registration', 'event-tickets' ) );
+		return apply_filters( 'tribe_tickets_attendee_registration_page_title', $title );
```
Comment (sc0ttkclark) [inline src/Tribe/Attendee_Registration/Template.php]:
```suggestion
		return apply_filters( 'tribe_tickets_attendee_registration_page_title', esc_html( $title ) );
```

File: src/Tribe/Attendee_Registration/Main.php [hunk 5]
```diff
@@ -42,16 +42,16 @@ public function get_slug() {
 		$slug = $page ? $page->post_name : '';
 
 		if (
-			empty( $slug )
+			! empty( $slug )
 			|| (
-				! empty( $page )
-				&& ! has_shortcode( $page->post_content, 'tribe_attendee_registration' )
+				empty( $page )
+				&& has_shortcode( $page->post_content, 'tribe_attendee_registration' )
```
Comment (sc0ttkclark) [inline src/Tribe/Attendee_Registration/Main.php]:
```suggestion
				|| has_shortcode( $page->post_content, 'tribe_attendee_registration' )
```

File: src/Tribe/Attendee_Registration/Main.php [hunk 6]
```diff
@@ -42,16 +42,16 @@ public function get_slug() {
 		$slug = $page ? $page->post_name : '';
 
 		if (
-			empty( $slug )
-			|| (
-				! empty( $page )
-				&& ! has_shortcode( $page->post_content, 'tribe_attendee_registration' )
+			! empty( $slug )
+			&& (
+				empty( $page )
```
Comment (sc0ttkclark) [inline src/Tribe/Attendee_Registration/Main.php]:
```suggestion
				! empty( $page )
```

File: src/Tribe/Attendee_Registration/Main.php [hunk 7]
```diff
@@ -42,16 +42,16 @@ public function get_slug() {
...
+				|| has_shortcode( $page->post_content, 'tribe_attendee_registration' )
```
Comment (sc0ttkclark) [inline src/Tribe/Attendee_Registration/Main.php]:
```suggestion
				&& has_shortcode( $page->post_content, 'tribe_attendee_registration' )
```

**Pull request merged**